src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
```

Comment:
As a Java developer, I want the REST documentation that tells me what the client library is really doing. All too often the Java client documentation is incomplete or inaccurate.